spec/intl.html

```diff
@@ -1401,11 +1401,66 @@ <h1>
+              For *"constrain"*, values that do not form a valid date are clamped to the correct range.
+              It then returns an ISO Date Record with the corresponding ISO date.
             </dd>
           </dl>
+          <p>
+            The operation throws a *TypeError* exception if the own data properties present on _fields_ are insufficient to calculate a date in _calendar_.
+            For example, in calendars that use eras, a *TypeError* is thrown if _fields_'s properties do not include one of the following minimum combinations:
+          </p>
+          <ul>
+            <li>*"year"*, *"monthCode"*, *"day"*</li>
+            <li>*"year"*, *"month"*, *"day"*</li>
+            <li>*"era"*, *"eraYear"*, *"monthCode"*, *"day"*</li>
+            <li>*"era"*, *"eraYear"*, *"month"*, *"day"*</li>
+          </ul>
+          <p>
+            The operation throws a *RangeError* exception if the date described by _fields_ does not exist, and _overflow_ is *"reject"*; or if the date described by _fields_ forms a date outside the range allowed by ISODateTimeWithinLimits.
+          </p>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendarmonthdaytoisoreferencedate" type="abstract operation">
+          <h1>
+            CalendarMonthDayToISOReferenceDate (
+              _calendar_: a String,
+              _fields_: an ordinary Object with one or more own data properties,
+              _overflow_: a String,
+            ): either a normal completion containing a Record with fields [[ISOMonth]], [[ISODay]], and [[ReferenceISOYear]], or an abrupt completion
+          </h1>
+          <dl class="header">
+            <dt>description</dt>
+            <dd>
+              It performs implementation-defined processing to convert _fields_, that describes a calendar date without a year (i.e., month code and day pair, or equivalent) in the calendar represented by _calendar_, to a reference date in the ISO calendar.
+              It performs the overflow correction given by _overflow_ on the date described by _fields_ in order to arrive at a valid month code and day pair in _calendar_.
+              For *"reject"*, values that do not form a valid date cause an exception to be thrown, as described below.
+              For *"constrain"*, values that do not form a valid date are clamped to the correct range.
+              It then returns a Record with a reference ISO date that corresponds to that month code and day pair in a reference year as described below.
+            </dd>
+          </dl>
+          <p>
+            The fields of the returned Record form a reference date in the ISO calendar that, when converted to the calendar given by _calendar_, corresponds to the given month code and day pair in an arbitrary but deterministically chosen reference year.
+            The reference date is the latest ISO date corresponding to the calendar date, that is earlier than or equal to the ISO date December 31, 1972.
```

Comment:
Your wording here is really good. One minor suggestion.

```suggestion
            The reference date is the latest ISO date corresponding to the calendar date, that is also earlier than or equal to the ISO date December 31, 1972.
```

Reply:
Well, it's half your wording too :smile:

Reply:
Yeah, but you made it much clearer. As usual! :-)

Reply:
BTW, ideally we'd have at least one test for a calendar where there are two possible dates in ISO year 1972 in order to test whether implementations are choosing the later one instead of the earlier date.